app/uk/gov/hmrc/fileupload/write/envelope/messages.scala

```diff
@@ -212,11 +212,13 @@ object EventSerializer {
 
 // error
 
+trait ConflictingCommand
```

Comment:
This trait is created to identify conflicting commands on which we can do a re-publish